modules/retail/pages/retail-terminal-ratelimiting.adoc

```diff
@@ -0,0 +1,18 @@
+[[retail.terminal-ratelimiting]]
+== Rate Limiting Terminals
+
+Salt is able to run commands in parallel on a large number of terminals.
+This can potentially create large amounts of load on your infrastructure.
+You can use rate-limiting parameters to control the load in your environment.
+
+For more information about rate limiting on terminals, see xref:salt:salt-rate-limiting.adoc[].
+
+
+
+===  Troubleshooting
+
+Sometimes when attempting to reboot a terminal after attempting to apply the Saltboot formula, the terminal will hang at the boot screen.
+This can be caused by a presence ping timeout value being set at a value that is too low.
```

Comment:
I'd suggest to state the exact log lines and file so that users can be sure that is the real problem.

Such references could be added here or (even better) in the presence ping section.

Reply:
The log files are already mentioned in the presence ping section.